theory-test-start-admission/lambda.js

```diff
@@ -1,5 +1,34 @@
+require('dotenv').config();
+
+const AdmissionDAO = require('theory-test-admission-dao');
 const logger = require('logger');
-const main = require('./src/main');
+
+exports.handler = (event, context, callback) => {
+
+	// log inbound event
+	logger.debug('Received event: ', JSON.stringify(event));
+	logger.info('Running start admission');
+	let response;
+	if (event.DrivingLicenceNumber && event.AdmissionId) {
```

Comment:
And if not...?
There's no point doing validation if you aren't going to act when validation fails.

Reply:
done